pages/portfolio.html

```diff
@@ -1,12 +1,101 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+    <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Portfolio | Grace Park</title>
+      <link href="../stylesheets/portfolio.css" rel="stylesheet">
+    </head>
+
+    <body class="container">
+
+      <header><h1 class="header-title">Grace Park</h1></header>
+
+      <nav class = "nav"> 
+        <a href="index.html">Home</a> | 
+        <a href="about.html">About</a> |
+        <a href="portfolio.html">Portfolio</a>
+      </nav>
+      
+      <main>
+          <h2>Portfolio</h2>
+
+        <section class="project-container">
+          <h2 class="project-title">Task List API</h2>
+          <p class="project-image"><img class="project-icon" alt="Tasklist checkmark icon" src="/assets/tasklist.png"/></p>
+          <p class="project-description"> A RESTful web API that allows a client to create, read, update, and delete tasks. Tasks are able to be sorted and calls Slack's API to post task updates on a Slack channel. 
+            <br>
+            <br>
+            Built with: Python, Flask, Postman, Postgres, SQLAlchemy, Heroku
+            <br>
+            <a href="https://github.com/gpark439/task-list-api">Github Repo</a>
+          </p>
+        </section>
+
+        <section class="project-container">
+          <h2 class="project-title">Layla's Chicken Nuggie Challenge</h2>
+          <p class="project-image"><img class="project-icon" alt="Layla's Chicken Nuggie Challenge dog icon" src="/assets/dog.png"/></p>
+          <p class="project-description">An interactive game featuring Layla, a chubby dog who tries to sneak chicken nuggies while her owner isn't looking. 
+            <br>
+            <br>
+            Built with: MIT's Scratch
+            <br><a href="https://scratch.mit.edu/projects/524056233">Play the game</a>
+          </p>
+        </section>
```

Comment:
This was actually really fun! I'm going to have that song stuck in my head all day now 😂